$NetBSD$

Adjust NAME and SYNOPSIS in the manual page for the installed
"openocd-rpi" program name.

--- doc/openocd.1.orig	2026-03-18 14:32:57.000000000 +0000
+++ doc/openocd.1
@@ -1,9 +1,9 @@
-.TH "OPENOCD" "1" "June 18, 2024"
+.TH "OPENOCD-RPI" "1" "June 18, 2024"
 .SH "NAME"
-openocd \- A free and open on\-chip debugging, in\-system programming and
+openocd\-rpi \- A free and open on\-chip debugging, in\-system programming and
 boundary\-scan testing tool for microcontrollers and other embedded devices
 .SH "SYNOPSIS"
-.B openocd \fR[\fB\-fsdlchv\fR] [\fB\-\-file\fR <filename>] [\fB\-\-search\fR <dirname>] [\fB\-\-debug\fR <debuglevel>] [\fB\-\-log_output\fR <filename>] [\fB\-\-command\fR <cmd>] [\fB\-\-help\fR] [\fB\-\-version\fR]
+.B openocd\-rpi \fR[\fB\-fsdlchv\fR] [\fB\-\-file\fR <filename>] [\fB\-\-search\fR <dirname>] [\fB\-\-debug\fR <debuglevel>] [\fB\-\-log_output\fR <filename>] [\fB\-\-command\fR <cmd>] [\fB\-\-help\fR] [\fB\-\-version\fR]
 .SH "DESCRIPTION"
 .B OpenOCD
 is an on\-chip debugging, in\-system programming and boundary\-scan
